frontend/src/components/Banner.jsx

```diff
@@ -32,6 +32,7 @@ const Banner = ({ setFilterOption, setNewsData }) => {
       })
       .then((data) => {
         setNewsData(data);
+        setLoading(false); // no longer loading now that there is data
```

Comment:
As-is if there was an error in the backend, the loading state will permanently stay true. We should add this into the catch block as well